t/22-dashboard.t

```diff
@@ -100,8 +101,15 @@ like(
 );
 
 $t->get_ok('/dashboard_build_results')->status_is(200);
-@h2 = $t->tx->res->dom->find('h2 a')->map('text')->each;
-is_deeply(\@h2, ['opensuse test', 'Test parent'], 'parent group shown and opensuse is no more on top-level');
+@h2 = $t->tx->res->dom->find('h2 a')->map(sub ($e) { $e->text . ($e->attr('title') // '') })->each;
+is_deeply(\@h2, ['opensuse test', 'Test parent', 'Shows the latest test results for all job groups within this parent job group'], 'parent group shown and opensuse is no more on top-level');
+$t->element_exists('#test_result_overview_link_1');
+my $tests_overview_dashboard = $t->tx->res->dom->find("#test_result_overview_link_1")->first;
+is(
+    $tests_overview_dashboard->attr("href"),
+    "/tests/overview?todo=1&groupid=1001",
```

Comment:
Prefer single quotes if no interpolation is needed.

Reply:
done